### DISCUSSION TITLE
<question>

### DISCUSSION THREAD

**Discussion by Lepus123** [community/community#40684] [New to GitHub]
### Select Topic Area

Question

### Body

Dear Github community.
i don't understeand the tremens of service.
so I have a question: 
Is it Legal to public a sales website.
Lepus123.

**Comment by DSDmark** [comment]
Hi @Lepus123,

if you're talking about publish sales website on **github-pages**. It means you want to use GitHub commercially. So, you can use commercial purposes as long as its usage remains within GitHub [terms of service](https://help.github.com/articles/github-terms-of-service/#e-private-repositories).

**Comment by airtower-luna** [comment]
If you mean using Pages as @DSDmark guessed, the answer is that the Pages documentation says this under ["Prohibited Uses"](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#prohibited-uses):

> GitHub Pages is not intended for or allowed to be used as a free web-hosting service to run your online business, e-commerce site, or any other website that is primarily directed at either facilitating commercial transactions or providing commercial software as a service (SaaS).

If you mean having a repository for your website code while the actual site is hosted elsewhere that's an entirely different matter and shouldn't be an issue.

**Comment by ghostinhershell** [comment]
Welcome to the GitHub Community, @Lepus123, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
